src/include/stir/Bin.h

```diff
@@ -91,8 +87,6 @@ class Bin
 private :
   // shared_ptr<ProjDataInfo> proj_data_info_ptr; 
   
-  int  segment;
```

Comment:
shouldn't we have a viewgram_indices member as well?

Reply:
it's derived from it.